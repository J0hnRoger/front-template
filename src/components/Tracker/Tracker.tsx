import * as React from 'react';

class Tracker extends React.Component {
  render() {
   return (
    <div className="card">
      <h4 className="placehold__title">Tracker</h4>
    </div>
   );
 }
}

export default Tracker;