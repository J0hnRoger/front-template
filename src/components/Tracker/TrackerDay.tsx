import * as React from 'react';

class TrackerDay extends React.Component {
  render() {
   return (
    <div className="tracker__item">
        1
    </div>
   );
 }
}

export default TrackerDay;
