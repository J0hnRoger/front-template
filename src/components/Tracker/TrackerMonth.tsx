import * as React from 'react';

interface Props {
    month: any;
}

class TrackerMonth extends React.Component<Props> {
  render() {
   return (
    <div className="tracker__month">
    </div>
   );
 }
}

export default TrackerMonth;
