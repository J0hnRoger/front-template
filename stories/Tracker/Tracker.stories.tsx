import * as React from 'react';
import { storiesOf } from '@storybook/react';

import withTests from '../../.storybook/withTests';

import Tracker from '@components/Tracker';

const stories = storiesOf('Tracker', module)
  .addDecorator(withTests('Tracker'));

stories
.add('default', () => (<Tracker />));