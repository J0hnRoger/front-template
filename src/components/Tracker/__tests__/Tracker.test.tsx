import * as React from 'react';
import { shallow } from 'enzyme';

import Tracker from '../';

/**
  * Tests List
  *
  */
const setup = (propOverrides) => {
   const props = Object.assign({
   }, propOverrides);

   const wrapper = shallow(<Tracker {...props} />);

   return {
      props,
      wrapper,
      clear: wrapper.find('.clear-completed'),
      count: wrapper.find('.todo-count'),
   };
};

describe('Tracker', () => {
   test('Tracker s\'affiche correctement', () => {
      const result = setup({})
        expect(result.wrapper).toMatchSnapshot();
   });
})
