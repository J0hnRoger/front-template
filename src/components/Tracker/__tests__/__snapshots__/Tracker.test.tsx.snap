// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tracker Tracker s'affiche correctement 1`] = `
<div
  className="card"
>
  <h4>
    Tracker
  </h4>
</div>
`;
